### PR TITLE
Update Party Spells

### PR DESCRIPTION
**Pull request by raymondtfr**
I have updated indentation, fixed duplicated ``subId``, simplified some expressions, replaced name of local variable ``tmp`` with ``count`` (a bit better name, as it refers to the affected players list), replaced ``FALSE`` with ``false``, added a missing condition parameter and fixed values of existing ones based on wikia.

### CODE REVIEW

File: data/spells/scripts/party/enchant.lua [hunk 1]
```diff
@@ -10,9 +10,9 @@ condition:setParameter(CONDITION_PARAM_STAT_MAGICPOINTS, 1)
 condition:setParameter(CONDITION_PARAM_BUFF_SPELL, true)
 
 local baseMana = 120
```
Comment (ranisalt) [inline data/spells/scripts/party/enchant.lua]: You can actually move this var inside the function, it's not used anywhere else.

Comment (raymondtfr) [inline data/spells/scripts/party/enchant.lua]: But if I move it into ``onCastSpell()`` won't it be created every time the spell is cast?

Comment (ranisalt) [inline data/spells/scripts/party/enchant.lua]: Hmm yeah it will, though I don't see if it is a performance hit at all. Keep it that way actually.

**Comment by marksamman** [comment]
Thanks!

**Pull request merged**